manifest.samples.json

```
@@ -90,56 +96,56 @@
           },
           "platforms": [
             {
-              "dockerfile": "samples/aspnetapp/Dockerfile.alpine-slim",
+              "dockerfile": "samples/aspnetapp/Dockerfile.chiseled",
               "os": "linux",
               "osVersion": "alpine",
               "tags": {
-                "aspnetapp-alpine-slim-amd64": {}
+                "aspnetapp-alpine-amd64": {}
```

```suggestion
                "aspnetapp-jammy-chiseled-amd64": {}
```
Same for the other aspnetapp image specific tags.

---

Consider changing `jammy` to `ubuntu` as this is an appliance image so it really doesn't matter what version of Ubuntu this image is. The same change was done for .NET Monitor so that if chiseled images for the next Ubuntu LTS are made available, we aren't forcing folks to change from `dotnet/monitor:8-jammy-chiseled` to `dotnet/monitor:9-<codename>-chiseled` when upgrading to .NET 9 images.

---

Oops! Fixed that up.

I think that the official names make more sense for our samples since they are intended as an extension of guidance and have a high-fidelity relationship back to the product repos. I think `monitor` has more flexibility on naming since they are appliance like. If you want to call them `ubuntu-chiseled`, that works for me. I just means that you need to stick to one Ubuntu version per Monitor version, which you likely want to do already.